Week 01/assignment/index.html

```diff
@@ -0,0 +1,110 @@
+    <title>Sopt Web</title>
+  </head>
+  <body>
+    <section class="home">
+      <!-- 헤더 -->
+      <header class="header">
+        <nav class="header__left">
+          <h2 class="header__title">HappyGonni</h2>
+          <ul>
+            <li><a href="#">홈</a></li>
+            <li><a href="#infinite_bannner">동물사진 🐶</a></li>
+            <li><a href="#scroll_view">큐티뽀쨕 사진첩 👀</a></li>
+          </ul>
+        </nav>
+        <nav class="header__right">
+          <ul>
+            <li>
+              <a href="https://github.com/gonn-i?tab=repositories" aria-label="깃헙으로 이동"
+                ><i class="fa-brands fa-github"></i
+              ></a>
+            </li>
+            <li>
+              <a href="https://www.instagram.com/k._.9oni/" aria-label="인스타로 이동"
+                ><i class="fa-brands fa-instagram"></i
+              ></a>
+            </li>
+            <li>
+              <a href="https://velog.io/@gonn-i/posts" aria-label="벨로그로 이동"><i class="fa-solid fa-pen"></i></a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+      <!-- 홈배너 -->
+      <img class="home_banner__img" alt="Home banner image" src="./assets/home_banner.jpeg" />
+      <div class="home_banner__etc">
+        <span>
+          <p>AND SOPT Wave Web 🌊</p>
+        </span>
+        <a href="https://github.com/AND-SOPT-WEB" aria-label="앤솝 웹 깃헙으로 이동" class="home_banner__button">
+          <i class="fa-solid fa-arrow-up-right-from-square"></i>
+          <p>바로가기</p>
+        </a>
+      </div>
+    </section>
+    <main>
+      <!-- 무한 배너 -->
+      <section id="infinite_bannner" class="infinite_banner">
+        <h2 class="banner_list--title">동물사진 모음 zip 🐶</h2>
+        <section class="infinite_banner__photos">
+          <ul class="banner__list l1">
+            <li><img alt="happy dog" src="./assets/pic1.jpeg" /></li>
+            <li><img alt="cute dochi" src="./assets/pic2.jpeg" /></li>
+            <li><img alt="cute dog" src="./assets/pic3.jpeg" /></li>
+            <li><img alt="blind dog" src="./assets/pic4.jpeg" /></li>
+            <li><img alt="guest dog" src="./assets/pic5.jpeg" /></li>
+            <li><img alt="annoying dog" src="./assets/pic6.jpeg" /></li>
+          </ul>
+          <ul class="banner__list l2">
+            <li><img alt="happy dog" src="./assets/pic1.jpeg" /></li>
+            <li><img alt="cute dochi" src="./assets/pic2.jpeg" /></li>
+            <li><img alt="cute dog" src="./assets/pic3.jpeg" /></li>
+            <li><img alt="blind dog" src="./assets/pic4.jpeg" /></li>
+            <li><img alt="guest dog" src="./assets/pic5.jpeg" /></li>
+            <li><img alt="annoying dog" src="./assets/pic6.jpeg" /></li>
+          </ul>
+        </section>
+      </section>
+      <!-- 스크롤 뷰 -->
+      <section id="scroll_view" class="scroll_view">
+        <h2 class="scroll_view--title">마음을 사르르 녹일 큐티뽀쨕 Top 10 👀</h2>
+        <section class="scroll_view__photos">
```

Comment:
```suggestion
        <section class="scroll-view__photos">
```
BEM방법론에서 일반적으로 여러단어의 조합은 _보다는 -로 연결하여 작명하는 것으로 알고있습니다.
이건 굳이 다 수정하실 필요는 없을 것 같습니다!

Reply:
단어의 연결은 `-` ! 확인했습니다